pipelines/managed/release-to-mrrc/README.md

```diff
@@ -19,3 +19,4 @@ Tekton release pipeline to release maven artifacts extracted from the maven repo
 | verify_ec_task_git_revision     | The git revision to be used when consuming the verify-conforma task                                                                | No       | -                                                         |
 | taskGitUrl                      | The url to the git repo where the release-service-catalog tasks to be used are stored                                              | Yes      | https://github.com/konflux-ci/release-service-catalog.git |
 | taskGitRevision                 | The revision in the taskGitUrl repo to be used                                                                                     | No       | ""                                                        |
+| releasePath                     | Path to the JSON string of the Release CR                                                                                          | No       | -                                                         |
```

Comment:
This is the README of the pipeline. But from what I can see, this param is being added to the collect-mrrc-params task, not the pipeline.